src/test/v2.0/advanced/channels-support-2.0.ts

```diff
@@ -108,13 +108,17 @@ export class ChannelControl2_0 implements ChannelControl<Channel, Context> {
 
   setupAndValidateListener1 = async (channel: Channel, expectedContextType: string, errorMessage: string, onComplete: (ctx: Context) => void) => {
```

Comment:
See suggestion on 1.2 version of this function. We *have* to separate the requested context type and expected context type to make this work